src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -302,21 +302,8 @@ protected function getConstructor(array &$data, $class, array &$context, \Reflec
      *
      * @throws RuntimeException
      */
-    protected function instantiateObject(array &$data, $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes/*, $format = null*/)
+    protected function instantiateObject(array &$data, $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes, ?string $format = null)
```

Comment:
@nicolas-grekas I agree with you on that one that there's no semantic difference here, but that's only the case as long as the default value is `null`. Even if redundant, the ability to change the default value without BC break is compelling.